ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -0,0 +1,196 @@
+# This is continuous link flap test. In this test,
+# 1.) Flap all interfaces one by one in 1-3 iteration
+# to cause BGP Flaps.
+# 2.) Flap all interfaces on peer (FanOutLeaf) one by one 1-3 iteration
+# to cause BGP Flaps.
+# 3.) Watch for memory (show system-memory) ,orchagent CPU Utilization
+# and Redis_memory.
+# Pass Criteria: All routes must be re-learned with < 5% increase in Redis
+# and ORCH agent CPU consumption below 10% after 3 mins after stopping flaps.
+
+- block:
+    - name: Record  memory status at start
```

Comment:
>  [](start = 19, length = 1)

One extra blank char.

Reply:
b/w Record and memory right ?, sure I will address it.